bpf-oci/assets/docker-config-test.json

```diff
@@ -0,0 +1,7 @@
+{
+    "auths": {
+        "ghcr.io": {
+            "auth": "YWFhOmJiYg=="
```

Comment:
is it hard coded the auth here?

Reply:
This file if for testing. The authencatin string is aaa:bbb